content/docs/onpremise/_index.md

```diff
@@ -0,0 +1,5 @@
++++
+title = "Kubeflow on-Premises"
```

Comment:
Let's standardize the capitalization as "On-premises" when used in a page title.